paderbox/array/interval/core.py

```diff
@@ -358,6 +358,42 @@ def _intervals_as_str(self):
         i_str = ', '.join(i_str)
         return i_str
 
+    @property
+    def as_serializable(self):
+        """
+        Exports intervals and length of ArrayInterval to a serializable object.
+        Reverting 
+        Allows easy export of ArrayIntervals, e.g. into .json format. 
+        >>> from IPython.lib.pretty import pprint
+        >>> from paderbox.array.interval.core import ArrayInterval
+        >>> ai = ArrayInterval.from_str('1:4, 5:20, 21:25', shape=50)
+        >>> ai
+        ArrayInterval("1:4, 5:20, 21:25", shape=(50,))
+        >>> ai.as_serializable
+        ('1:4, 5:20, 21:25', (50,))
+        """
+        assert self.inverse_mode is False, 'Export of intervals as tuple is only valid for normal mode, not inverse mode!' 
+        return self._intervals_as_str, self.shape
```

Comment:
Can you add a comment, that we use `self._intervals_as_str` instead of `intervals` to increase the human readability and reduce the size of the serialized object inside a human-readable format like json?

Reply:
Is there an easy way to convert an array with `inverse_mode=True` into one with `inverse_mode=False`? If so, it would be good to have a comment about this in the docstring or error message.

Reply:
It depends, when the shape is known, you can convert it. Otherwise, it is not possible, i.e. the last interval goes to infinity.